components/survey/element.js

```diff
@@ -70,20 +66,16 @@ export class MDNSurvey extends L10nMixin(LitElement) {
    * @returns {Survey.Survey | undefined}
    */
   #findSurvey() {
+    const forcedSurvey = new URLSearchParams(location.search).get(
```

Comment:
What was the reason to move from hash to search?

```suggestion
    const forcedSurvey = new URLSearchParams(location.hash).get(
```